libs/ui/src/types/formAndField.ts

```diff
@@ -36,14 +36,17 @@ export type PrimitiveSizable = {
 };
 
 export type PrimitiveAddons = {
-  rightAddon?: React.ReactNode
-}
+  rightAddon?: React.ReactNode;
+};
```

Comment:
Thinking we might not want to extend right add on to all fields?

Reply:
This is only a formatting change change, there should be changed from what already exists

Reply:
> This is only a formatting change change, there should be changed from what already exists

ah whoops. got it.

Reply:
Let's figure out where that got put in and address it.

Reply:
> Let's figure out where that got put in and address it.

@jordanlesich 

It was added with this one that says to add it to fieldwrapper:
https://github.com/HausDAO/daohaus-monorepo/issues/568

Where would you rather see it? I'll open a new ticket to move it.

Reply:
Oh, it's just for the wrapper. I thought it was for the input. My bad. This is exactly what we want.